### ISSUE TITLE
Docker for Airflow 1.10.6

### ISSUE THREAD

**Issue by mikolajmorawski** [datarods/docker-airflow#2]
Hi, 
 First of all, great idea with this fork from "puckel" repo to this new repository. Thank you 😊.
So I am creating this first issue, so we can plan to extend the airflow to 1.10.6 version because it was released 2 days ago.
Regards,
Mikolaj

**Comment by impredicative** [comment]
Hi. We would need:
(1) a PR, probably based off an open PR for 1.10.5 for the puckel repo.
(2) 3 reviewers which I think we can get.

**Comment by mikolajmorawski** [comment]
That's right, but I think the 1.10.5 version should be released first here then we can create this PR.
Regards,
Mikolaj
